src/main/java/org/ibp/api/java/impl/middleware/dataset/DatasetCSVExportServiceImpl.java

```diff
@@ -21,13 +21,20 @@
 import javax.annotation.Resource;
 import java.io.File;
 import java.io.IOException;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
 import java.util.stream.Collectors;
 
 @Service
 @Transactional
 public class DatasetCSVExportServiceImpl extends AbstractDatasetExportService implements DatasetExportService {
+
 	static final String LOCATION_ID_VARIABLE_NAME = "LOCATION";
```

Comment:
Can we reuse the same location related constants in DatasetServiceImpl instead of duplicating here?